src/container/History/index.tsx

```diff
@@ -26,15 +29,19 @@ const History = () => {
         walletStore: { puzzleHash, isAblyConnected },
     } = rootStore
 
-    const groupedHistory = useMemo(
-        () =>
-            groupBy(history, (item) =>
-                formatDistanceToNowStrict(item.updatedAt, {
-                    addSuffix: true,
-                })
-            ),
-        [history]
-    )
+    const groupedHistory = useMemo(() => {
+        const option: { addSuffix: boolean; locale?: any } = {
+            addSuffix: true,
+        }
+
+        if (i18n.language !== 'en') {
+            option.locale = zhCN
+        }
+
+        return groupBy(history, (item) =>
+            formatDistanceToNowStrict(item.updatedAt, option)
+        )
+    }, [history, i18n.language])
```

Comment:
here does not need depend `i18n.language` , because it is not in the react lifecycle

Reply:
or. change to const { t, i18n } = useTranslation()
and default `option.locale` = zhCN use` en`